src/tests/cmocka/test_nss_srv.c

```diff
@@ -1800,6 +1800,8 @@ void test_nss_getgrnam_mix_dom_nonfqnames(void **state)
 {
     errno_t ret;
 
+    nss_test_ctx->subdom->output_fqnames = false;
+
```

Comment:
If there is no way to set this parameter to false for a user, do we need to set it in the tests? Shouldn't the test be rather fixed or even removed?